src/components/timers/Countdown.tsx

```diff
@@ -1,3 +1,62 @@
-const Countdown = () => null;
+import React, { useState, useEffect } from 'react';
 
-export default Countdown;
+const Countdown: React.FC = () => {
+  // track remaining time in sec
+  const [time, setTime] = useState(150); // deliverable eg: starting from 2 min 30 sec
+  const [isRunning, setIsRunning] = useState(false);
+
+ // useEffect handle timer interval when timer is running
+  useEffect(() => {
+    if (isRunning && time > 0) {
+      // decrement time by 1 sec
+      const interval = setInterval(() => {
```

Comment:
your interval is being recreated every second. Intervals are functions that repeat periodically, so if you're going to reset it every second its better to use setTimeout, which executes only once.